kahuna/public/js/search/results.js

```diff
@@ -146,6 +151,8 @@ results.controller('SearchResultsCtrl', [
             ctrl.imagesAll = [];
             ctrl.imagesAll.length = Math.min(images.total, ctrl.maxResults);
 
+            imagesPositions = new Map;
```

Comment:
Meh, we enforce parens. Vaguely convinced by [the explanation](https://jslinterrors.com/missing-invoking-a-constructor), so will add them here.